metadata_backend/api/middlewares.py

```diff
@@ -47,23 +50,98 @@ async def http_error_handler(req: Request, handler: Callable) -> Response:
 @middleware
 async def check_login(request: Request, handler: Callable) -> StreamResponse:
     """Check login if there is a username."""
-    if request.path not in ["/", "/aai", "/callback"] and "OIDC_URL" in os.environ and bool(os.getenv("OIDC_URL")):
-        session = await get_session(request)
-        token = session.get("access_token")
-        logged = request.cookies.get("logged_in")
-        if not (token and logged):
+    if (
+        request.path
+        in [
+            "/schemas",
+            "/drafts",
+            "/validate",
+            "/submit",
+            "/folders",
+            "/objects",
+            "/users",
+            "/logout",
+            "/home",
+            "newdraft",
```

Comment:
Should this instead be?
```suggestion
            "/newdraft",
```

Reply:
good catch